examples/fcma/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mpi4py
+sklearn
```

Comment:
Only list packages that are not already listed in `setup.py`. In this case, there is no need for this requirements file.